### ISSUE TITLE
Add Aarch64 as a fully supported architecture

### ISSUE THREAD

**Issue by SueChaplain** [eclipse/openj9-docs#732]
**Issue or pull request number:**
Various

**Overview:**
Aarch64 has so far been an early release and is expected to be fully supported for 0.26.0. We need to sweep the documentation and confirm that any differences are correctly called out:

- command line (-X, -XX, -D) option support (are all Linux options supported on Aarch64)
- which GC policies are supported
- known issues or limitations are documented (in the release notes, or in related topics)

**Release target:**
Eclipse OpenJ9 0.26.0

**Applies to the following JDK versions:**
OpenJDK 11 and later

**Applies to the following platforms:**
Linux

**Comment by SueChaplain** [comment]
@knn-k & @0xdaryl - is this work something that you can do between you? Please update this issue with any changes that you see are needed.

**Comment by 0xdaryl** [comment]
Yes, we will do that.  This is now targeted for JDK 17 GA.  I created an issue in the OpenJ9 repo to track it under the "AArch64 GA Release" project.
